cartographer/common/time.h

```diff
@@ -32,6 +32,12 @@ struct UniversalTimeScaleClock {
   using duration = std::chrono::duration<rep, period>;
   using time_point = std::chrono::time_point<UniversalTimeScaleClock>;
   static constexpr bool is_steady = true;
+
+  static time_point now() noexcept {
+    using namespace std::chrono;
+    return time_point(
+        duration_cast<duration>(system_clock::now().time_since_epoch()));
```

Comment:
This needs a correction for the different epochs. We should probably pull the kUtsEpochOffsetFromUnixEpochInSeconds constant from cartographer_ros's time_conversion.h over here.